api/src/main/java/jakarta/json/bind/customization/JsonbCustomizationBuilder.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package jakarta.json.bind.customization;
+
+import java.text.DateFormat;
+import java.text.NumberFormat;
+import java.util.Locale;
+
+import jakarta.json.bind.adapter.JsonbAdapter;
+import jakarta.json.bind.serializer.JsonbDeserializer;
+
+/**
+ * Common interface for all customization builders.
+ */
+public interface JsonbCustomizationBuilder<T extends JsonbCustomizationBuilder<T, B>, B> {
+
+    /**
+     * Whether this component can be nillable.
+     *
+     * @param nillable nillable component
+     * @return updated builder instance
+     */
+    T nillable(boolean nillable);
```

Comment:
How do you set the scope to nillable?

Reply:
OH ... thank you :-) completely missed that!